### PR TITLE
add api update for pickup parcel (bulk) weight changes

### PR DESCRIPTION
**Pull request by EirikStudsrodOlsen**

### CODE REVIEW

**Comment by WebDevBring** [comment]
[trello.com/c/6QmKFLne](https://trello.com/c/6QmKFLne)

File: content/api/revision-history/booking/2024-02-28.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: Booking API - Changes in weight limit for Pickup Parcel (0340) and Pickup Parcel Bulk (0342) to Denmark
+publishDate: 2024-02-28
+layout: api
+notanapi: true
```
Comment (tomgag) [inline content/api/revision-history/booking/2024-02-28.md:5]: notanapi: true I don't know what this is for, but this is an API

**Review by tomgag — APPROVED** [review]
LGTM

File: content/api/revision-history/booking/2024-02-28.md (hunk 1 above)
Comment (EirikStudsrodOlsen) [reply content/api/revision-history/booking/2024-02-28.md]: I thought i had set this to false! Good catch.

**Pull request merged**